samples/cellular/modem_shell/README.rst

```diff
@@ -929,12 +929,25 @@ LED indications
 
 The LEDs have the following functions:
 
-LED 1 (nRF91 Series DKs)/Purple LED (Thingy:91):
-   Lit for five seconds when the current location has been successfully retrieved by using the ``location get`` command.
-LED 2 (nRF91 Series DKs):
-   Indicates the state of the GPIO pin when pulse counting has been enabled using the ``gpio_count enable`` command.
-LED 3 (nRF91 Series DKs)/Blue LED (Thingy:91):
-   Indicates the LTE registration status.
+nRF91 Series DKs:
+
+   * **LED 2** indicates the state of the GPIO pin when pulse counting has been enabled using the ``gpio_count enable`` command.
+   * **LED 3** indicates the LTE registration status.
+   * **LED 4** is lit for five seconds when the current location has been successfully retrieved by using the ``location get`` command.
+
+Thingy:91 and Thingy:91 X:
+
+   * **RGB LED:**
```

Comment:
@peknis Do we need this RGB LED level here or could it be combined with the "Thingy:91 and Thingy:91 X:" line.

Reply:
Well, that is what we had there at some point but it appeared again on its own line. I would be glad to get rid of an extra layer of bullets and have it on the same line with Thingies.

Reply:
ok, so Matti can fix this. He is doing his first rebases so might be that something has gone wrong there.

Reply:
I considered that myself. But maybe it's better to align this with the DK chapter, as we may add more LED functionalities for Thingy in the future.

Reply:
Ok, so now "RGB LED" back on the same line with "Thingy:91 and Thingy:91 X"